operators/understanding/olm-packaging-format.adoc

```diff
@@ -62,3 +56,26 @@ For instructions about creating file-based catalogs by using the `opm` CLI, see
 For reference documentation about the `opm` CLI commands related to managing file-based catalogs, see xref:../../cli_reference/opm/cli-opm-ref.adoc#cli-opm-ref[CLI tools].
 
 include::modules/olm-fb-catalogs-automation.adoc[leveloffset=+2]
+
+include::modules/olm-rukpak-about.adoc[leveloffset=+1]
+
+The following sections further detail the individual components of RukPak.
```

Comment:
I get that you're trying to introduce what the next module inclusions are, but it kind of looks like you're introducing the additional links that come next.

I don't know what the right answer is here, but just wanted to share what it looked like to me. Maybe it would help to call out the names of the components (Bundle/BundleDeployment/Provisioner)?

Reply:
Dropping it cuz it doesn't add anything really.